### DISCUSSION TITLE
recommended posts showing up even though i have "no recommended" enabled

### DISCUSSION THREAD

**Discussion by alexevah** [AprilSylph/XKit-Rewritten#1090] [Help Desk] [answered]
I'm using the latest version of Xkit (0.21.8), and the latest version of firefox (114.0.1). It started happening maybe a week ago or so
https://imgbox.com/PdHmC3Fz
https://imgbox.com/BbrqGgTJ

**Comment by AprilSylph** [comment, marked as answer]
Not sure why the XKit option isn't working, but you can turn off followed tag injections via https://www.tumblr.com/settings/dashboard.

EDIT: Looks like we specifically exclude recommendations that have Tumblr toggles.
